packages/material_duration_picker/lib/material_duration_picker.dart

```diff
@@ -2367,8 +2367,8 @@ class _TimePicker extends StatefulWidget {
 class _DurationPickerState extends State<_TimePicker> with RestorationMixin {
   Timer? _vibrateTimer;
   late DurationPickerMaterialLocalizations localizations;
-  final RestorableEnum<_DurationMode> _durationMode =
-  RestorableEnum<_DurationMode>(_DurationMode.hour, values: _DurationMode.values);
+  late final  RestorableEnum<_DurationMode> _durationMode =
```

Comment:
Nice thank you! Can you remove the additional space?
I could not push as your branch is not editable by maintainers to do a quick fix.
```suggestion
  late final RestorableEnum<_DurationMode> _durationMode =
```